data-source/jsonnet/england-wales/household/blocks/who-lives-here/primary_person_list_collector.jsonnet

```diff
@@ -50,7 +50,7 @@ local questionTitle = {
     title: questionTitle,
     definitions: [
       {
-        title: 'What does "usually live" mean?',
+        title: 'What does “usually live” mean?',
```

Comment:
How come these titles are changing, I thought this should only be changing values?

Reply:
The build was failing because this line was different in the .po files. I'm not sure how or why these got out of sync because I hadn't changed this text. So I ended up having to make this change to allow the build to pass.

Reply:
I think that changing these titles and updating the .po files would be what we'd really want? The only thing changing in this PR should really be that single answer value.

Reply:
I will close this PR and open another with a clean branch.